index.html

```diff
@@ -72,7 +72,7 @@ <h2>LET’S CONNECT</h2>
     <section id="work">
         <section class="project-section">
             <figure>
```

Comment:
- [ ] Kindly 😃 try to make the header here like the Figma Design as you see here the hamburger icon needed to move to the right side and the logo needed to move to the left side  to make it nice like the Figma 🚀💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/235682407-278b7e26-4560-49d4-b170-0105e7c3f3ca.png) |    ![image](https://user-images.githubusercontent.com/83360636/235682753-44ea5790-02f8-4802-9270-1032e01b7424.png)  |